### DISCUSSION TITLE
Dependency review should show updated from to

### DISCUSSION THREAD

**Discussion by codesimplicity** [community/community#14390] [Pull Requests]
The beta feature of a rich diff for a `pom.xml` currently shows changes like this:
```
 org.springframework.boot:spring-boot-dependencies updated to 2.5.12
```

For a review I may want to know what the previous version was to get an indication on the risk or further checks needed etc.

Another detail: I'd prefer a more condensed view, not a big table row for each dependency, but more like a normal text diff (could also use the known colors: "updated from <red>2.5.10</red> to <green>2.5.12</green>"

**Comment by storedbox** [comment]
I **very strongly** second this; the dependency review rich diff should display both the updated-from and updated-to version numbers.
